index.html

```diff
@@ -0,0 +1,101 @@
+                        <li><a href="about.html" class="mobile-a">Join</a></li>
+                        <li><a href="#" class="mobile-a">Sponsor</a></li>
+                        <li><a href="#" class="button mobile-a">CC Campaign</a></li>
+                    </ul>
+        </nav>
+    </header>
+<main class="main-container">
+    <h1 class="first-h1">Hello! Welcome!!!</h1>
+        <h1 class="sec-h1">Meanwood Farmers Cooperative Conference</h1>
+            <p class="page-par">We take pride in helping out farmers get farming inputs on time in readiness for the farming season. Food security for our nation is our top priority
+            </p>
+            <p class="bold">
+           (MON)~(FRI) FROM 08:00am to 5:00pm
+            </p>
+            <p class="bottom-text">@ Meanwood Chamba Valley</p>
+</main>
+    <div class="main-section">
+        <h1 class="main-section-h1">
+            Main Program
+        </h1>
+            <div class="orange-underline"></div>
+                <ul class="blocks">
+                    <li><i class="fas fa-anchor"></i>
+                        <h2>Seminars</h2>
+                        <p>welcome to our free Seminars everyday</p>
+                    </li>
+                    <li><i class="fas fa-anchor"></i>
+                        <h2>Lectures</h2>
+                        <p>Learn mordern farming techniques</p>
+                    </li>
+                    <li><i class="fas fa-anchor"></i>
+                        <h2>Gov Programs</h2>
+                        <p>Get subsidized farming inputs</p>
+                    </li>
+                    <li><i class="fas fa-anchor"></i>
+                        <h2>Assistance</h2>
+                        <p>Access technical support</p>
+                    </li>
+                </ul>
+            <a href="#" class="see-more">See more</a>    
+     </div>
+<section class="speaker-sec">
+    <h2 class="speaker-header">Featured Guests</h2>
+        <div class="orange-underline"></div>
+            <ul class="speaker-list" id="speaker-list">
+               
+            </ul>
+        <button class="more-btn" id="more-btn" type="button">More</button>
+</section>
+<section class="partner">
+    <h2 class="partner-header">Partners</h2>
+        <div class="orange-underline"></div>
+    <ul class="bottom-logos">
+        <li><a href=""><img src="img/ebay.png" alt="logo"></a></li>
+        <li><a href=""><img src="img/who.svg" alt="logo"></a></li>
+        <li><a href=""><img src="img/agriculture.png" alt="logo"></a></li>
+        <li><a href=""><img src="img/skype.png" alt="logo"></a></li>
+    </ul>
+        
+</section>
+
+<footer class="footer">
+    <img class="logo" src="img/MyLogo.png" alt="logo">
+        <div class="footer-text">
+            <span class="footer-copyright">2021 @Meanwood Farmers Cooperative conference </span>
+            <span class="footer-about">Diversification through agriculture.</span>
+        </div>
+</footer> 
```

Comment:
* I suggest you stick to the design template in this section as much as possible 
* Also the proper fonts, color and margins from the design template for this section are missing please fix it
Yours : 
![Screenshot (143)](https://user-images.githubusercontent.com/52497246/124363066-f6145280-dc30-11eb-9b51-61b13448bfec.png)
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------Original : 
![Screenshot (132)](https://user-images.githubusercontent.com/52497246/124363082-0c221300-dc31-11eb-8d7f-53e047f76a48.png)

Reply:
- [ ] Logo looks better now. It looks really good. But the margin that you put between items was not necessary. Please reduce that margin. 

![Screenshot 2021-07-04 at 02-31-26 Meanwood Farmers Cooperative Conference](https://user-images.githubusercontent.com/1642014/124369049-38a94f80-dc70-11eb-885f-80b677833b76.png)

It should look like this :arrow_double_down: 
![Screenshot 2021-07-04 at 02-33-10 Meanwood Farmers Cooperative Conference](https://user-images.githubusercontent.com/1642014/124369055-46f76b80-dc70-11eb-8431-6e1f9fb47f88.png)

Reply:
Thank you I have done this part